.github/workflows/covector-version-or-publish.yml

```
@@ -0,0 +1,46 @@
+name: version or publish
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  version-or-publish:
+    runs-on: ubuntu-latest
+    timeout-minutes: 65
+    outputs:
+      change: ${{ steps.covector.outputs.change }}
+      commandRan: ${{ steps.covector.outputs.commandRan }}
+      successfulPublish: ${{ steps.covector.outputs.successfulPublish }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: cargo login
+        run: cargo login ${{ secrets.crate_token }}
```

@nothingismagick I assume that means we need to switch this?

```suggestion
        run: cargo login ${{ secrets.ORG_CRATE_TOKEN }}
```